test/functional-test.fnl

```diff
@@ -47,4 +47,28 @@
         (fn []
           (is.eq? (f.reduce #(.. $1 $2) "" [5 4 3 2 1]) "54321" "reduce did not concat list into string")
           (is.eq? (f.reduce #(if (> $1 $3) $1 $3) 0 [1 3 5 2 0]) 5 "reduce did not find max")))
-    ))
+
+    (it "(map) traverses a table"
+        (fn []
+          (is.eq?
+           (fennel.view (f.map (fn [x] x) [:a :b :c]))
+           (fennel.view [:a :b :c])
+           "same table")))
+
+    (it "(map) traverses a table with a transform"
+        (fn []
+          (is.eq?
+           (fennel.view (f.map (fn [x] (string.upper x)) [:a :b :c]))
+           (fennel.view [:A :B :C])
```

Comment:
you can use `is.seq-eq` instead of using `fennel.view` to compare tables, as long as they don't contain tables.

Reply:
I converted the first two cases, but the last one I couldn't figure out. Not sure what's causing it to fail when I'm trying with seq-eq?

Reply:
That's because it has nested tables. No big deal. We can write a better seq-eq? later.